core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -666,6 +667,11 @@ public SystemSessionProperties(
                         INCREMENTAL_HASH_ARRAY_LOAD_FACTOR_ENABLED,
                         "Use smaller load factor for small hash arrays in order to improve performance",
                         featuresConfig.isIncrementalHashArrayLoadFactorEnabled(),
+                false),
```

Comment:
this indent looks weird

Reply:
Oops. Apologies for being clumsy of me. Done.